subiquity/ui/views/filesystem/tests/test_partition.py

```diff
@@ -129,8 +129,8 @@ def test_edit_existing_partition_mountpoints(self):
         partition = model.add_partition(disk, 512*(2**20))
         partition.preserve = True
         fs = model.add_filesystem(partition, "ext4")
-        fs.preserve = True
         model._orig_config = model._render_actions()
+        fs.preserve = True
```

Comment:
Odd change, does it matter?  if so why?

Reply:
This test is poking around in the guts of things for reasons I don't really remember :/

But the reason for this change is reasonably clear: we want the fs to be in _orig_config and now that render_actions does not include things with preserve = True we need to set that after calling it.